packages/flutter_tools/test/commands.shard/hermetic/doctor_test.dart

```diff
@@ -866,6 +868,65 @@ void main() {
       DoctorValidatorsProvider: () => FakeDoctorValidatorsProvider(),
       Analytics: () => fakeAnalytics,
     });
+
+    testUsingContext('contains installed and partial', () async {
+      await FakePassingDoctor(logger, clock: fakeSystemClock).diagnose(verbose: false);
+
+      expect(fakeAnalytics.sentEvents.length, 4);
```

Comment:
nit, but I think this will give a nicer failure message:

```suggestion
      expect(fakeAnalytics.sentEvents, hasLength(4));
```

Reply:
👍